docs/quick-start.md

```diff
@@ -59,5 +90,21 @@ You can also observe the logs of the Workflow run by running the following:
 argo logs -n argo @latest
 ```
 
+### Submit an example workflow (GUI)
+
+* Open a port-forward so you can access the UI:
+
+```bash
+kubectl -n argo port-forward deployment/argo-server 2746:2746
+```
+
+* Navigate your browser to <https://localhost:2746>.
```

Comment:
Remove `<>` so readers can just click the link?

Reply:
This breaks the CI. (This url was also written like this previously)

Reply:
The `>` chars are removed at rendering. Look at https://argoproj.github.io/argo-workflows/quick-start/